tests/test_models.py

```
@@ -0,0 +1,152 @@
+INDEX_NAME = "ix_webhooks_repo_pr_number_created_at"
+
+
+class TestWebhookCompositeIndex:
+    """Tests for the composite index on the Webhook model."""
+
+    @staticmethod
+    def _find_index(name: str) -> Index:
+        for arg in Webhook.__table_args__:
+            if isinstance(arg, Index) and arg.name == name:
+                return arg
+        raise AssertionError(f"Index {name} not found in __table_args__")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)